flink-libraries/flink-table/src/test/scala/org/apache/flink/table/runtime/batch/sql/AggregateITCase.scala

```diff
@@ -559,4 +559,41 @@ class AggregateITCase(
 
     TestBaseUtils.compareResultAsText(result.asJava, expected)
   }
+
+  @Test
+  def testMultipleDistinctWithDiffParams(): Unit = {
+    val env = ExecutionEnvironment.getExecutionEnvironment
+    val tEnv = TableEnvironment.getTableEnvironment(env, config)
+
+    val sqlWithNull = "SELECT a, " +
+      " CASE WHEN b = 2 THEN null ELSE b END AS b, " +
+      " c FROM MyTable"
+
+    val sqlQuery =
+      "SELECT b, " +
+      "  COUNT(DISTINCT b), " +
```

Comment:
Could you also add a `IS NOT DISTINCT FROM` expression test to `org.apache.flink.table.expressions.ScalarOperatorsTest`?